src/tt.cpp

```diff
@@ -48,6 +47,8 @@ void TranspositionTable::resize(size_t mbSize) {
                 << "MB for transposition table." << std::endl;
       exit(EXIT_FAILURE);
   }
+  memset(mem, 0, clusterCount * sizeof(Cluster) + CacheLineSize - 1);
```

Comment:
Yes the difference between mem and table is only cache line alignment.  We don't care what's in the first few bytes of mem that will never be accessed.